src/app/space/create/deployments/deployments-donut/deployments-donut.component.ts

```diff
@@ -24,6 +24,18 @@ export class DeploymentsDonutComponent implements OnInit {
   desiredReplicas: number;
   debounceScale = debounce(this.scale, 650);
 
+  colors = {
```

Comment:
Please use the PatternFly colors for all of these items and comment what variables they are. Example:
```
'Running': '#004368', // @color-pf-blue-600
```